SessionMessagingKit/Open Groups/OpenGroupAPI.swift

```diff
@@ -213,9 +215,10 @@ public enum OpenGroupAPI {
     /// Rooms to which the user does not have access (e.g. because they are banned, or the room has restricted access permissions) are not included
     public static func preparedRooms(
         authMethod: AuthenticationMethod,
-        using dependencies: Dependencies
+        using dependencies: Dependencies,
+        skipAuthentication: Bool = false
```

Comment:
`using dependencies` should always be the last parameter - please put `skipAuthentication` before it